glib/src/variant.rs

```diff
@@ -9,9 +9,18 @@
 //! `Variant` types are described by [`VariantType`](../struct.VariantType.html)
 //! "type strings".
 //!
-//! Although `GVariant` supports arbitrarily complex types, this binding is
-//! currently limited to the basic ones: `bool`, `u8`, `i16`, `u16`, `i32`,
-//! `u32`, `i64`, `u64`, `f64`, `&str`/`String`, and [`VariantDict`](../struct.VariantDict.html).
+//! Although `GVariant` supports arbitrarily complex types, this crate only provides bindings for a
```

Comment:
The caveat at the beginning is not really true anymore. We support arbitrarily complex types in the bindings now.